### PR TITLE
[Keyboard] Update Work Louder boards

### PR DESCRIPTION
**Pull request by drashna**
## Description

Update the Work Board to data driven config, as well as relocate via config, etc. 

## Types of Changes

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/work_louder/work_board/info.json [hunk 1]
```diff
@@ -4,120 +4,202 @@
...
+    },
+    "build": {
+        "lto": true
+    },
     "processor": "atmega32u4",
     "bootloader": "atmel-dfu",
+    "rgb_matrix": {
+        "driver": "WS2812",
+        "layout": [
+            { "flags": 1, "matrix": [3, 8], "x": 223, "y": 63 },
```
Comment (fauxpark) [inline keyboards/work_louder/work_board/info.json:42]: Probably should hold off on this, there are some issues with the way `qmk info` parses the `g_led_config`.

File: keyboards/work_louder/work_board/work_board.c [hunk 2]
```diff
@@ -136,4 +136,125 @@ void keyboard_pre_init_kb(void) {
...
+
+void eeconfig_init_kb(void) {
+    work_louder_config.raw       = 0;
+    work_louder_config.led_level = true;
+    eeconfig_update_kb(work_louder_config.raw);
+    eeconfig_init_user();
+}
+
+void keyboard_post_init_kb(void) {
+    work_louder_config.raw = eeconfig_read_user();
```
Comment (zvecr) [inline keyboards/work_louder/work_board/work_board.c:186]: _user?

Comment (zvecr) [inline keyboards/work_louder/work_board/work_board.c:186]: _user?

File: keyboards/work_louder/work_board/work_board.c [hunk 3]
```diff
@@ -136,4 +136,125 @@ void keyboard_pre_init_kb(void) {
...
+            *value_data = work_louder_config.led_level;
+            break;
+            // case id_wl_layer:
+            //     *value_data = get_highest_layer(layer_state);
+            //     break;
+    }
+}
+
+void wl_config_save(void) {
+    eeconfig_update_user(work_louder_config.raw);
```
Comment (zvecr) [inline keyboards/work_louder/work_board/work_board.c:227]: _user?

**Review by zvecr — CHANGES_REQUESTED** [review]
Mixture of user/kb eeprom calls

File: keyboards/work_louder/work_board/info.json (hunk 1 above)
Comment (zvecr) [reply keyboards/work_louder/work_board/info.json:42]: Why would putting this in info.json have anything to do with the parsing logic?

Comment (fauxpark) [reply keyboards/work_louder/work_board/info.json:42]: https://github.com/qmk/qmk_firmware/pull/19954

Comment (fauxpark) [reply keyboards/work_louder/work_board/info.json:42]: I can tell this was copied from `qmk info` output due to the key order and spacing around the braces.

Comment (drashna) [reply keyboards/work_louder/work_board/info.json:42]: Ah, yeah, it's off.  Fixed manually.

Comment (fauxpark) [reply keyboards/work_louder/work_board/info.json]: image
image
LGTM

File: keyboards/work_louder/work_board/info.json [hunk 4]
```diff
@@ -4,120 +4,202 @@
...
+        "rows": ["F0", "F1", "F4", "F5"]
+    },
     "encoder": {
       "rotary": [
         {"pin_a": "B0", "pin_b": "B1"}
       ]
     },
+    "features": {
+        "audio": false,
+        "backlight": false,
```
Comment (zvecr) [inline keyboards/work_louder/work_board/info.json:23]:
```suggestion
```

**Pull request closed without merging**